### PR TITLE
Add Go 1.12

### LINKED ISSUE #741: Update Golang to 1.12
**Tool infomation**
- Tool name: Golang
- Add or update? Update
- Desired version: 1.12

**Areas and ecosystems**

- [ ] Android
- [ ] C / C++
- [ ] Container tools
- [ ] Database servers or clients
- [ ] General purpose tools
- [X] Go
- [ ] Java
- [ ] Javascript, Node.js, and TypeScript
- [ ] iOS, macOS, and Apple-specific tools
- [ ] .NET Core or .NET Desktop
- [ ] PHP
- [ ] Python
- [ ] Ruby
- [ ] Rust
- [ ] Scala
- [ ] Testing and code coverage
- [ ] Xamarin
- [ ] Other

**Image impacted**
Add the appropriate label for the image or images that should be updated (e.g., mac, ubuntu, window).
Mac, Windows and Linux

### PR DESCRIPTION
**Pull request by DavidStaheli**
This fixes #741.  It adds Go 1.12 to the Windows and Linux images.

### CODE REVIEW

File: images/win/vs2017-Server2016-Azure.json [hunk 1]
```diff
@@ -533,8 +533,8 @@
         {
             "type": "powershell",
             "environment_vars": [
-                "GO_VERSIONS=1.9.7,1.10.8,1.11.5",
```
Comment (sachinma) [inline images/win/vs2017-Server2016-Azure.json]: Is it safe to assume people wont be using 1.9 versions in their tasks?

Comment (DavidStaheli) [reply images/win/vs2017-Server2016-Azure.json]: Hi @sachinma.  1.9.7 is archived but someone is still likely using it.  We're still forming the policy/guidance about when to remove old versions.  How about we keep 1.9.7 around until we have a better, established way to communicate upcoming changes (i.e. an RSS feed that people can follow)?

Comment (sachinma) [reply images/win/vs2017-Server2016-Azure.json]: I like the idea of having a policy and guidance around removing old software and a way to communicate it to the users. I agree till then it is probably safer to keep 1.9.7

Comment (DavidStaheli) [reply images/win/vs2017-Server2016-Azure.json]: @sachinma Go 1.9.7 is restored in this PR.

**Pull request merged**